site/en/docs/extensions/mv3/screen_capture/index.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Audio recording and screen capture"
+seoTitle: "Chrome Extensions: Audio recording and screen capture"
+date: 2023-04-14
+updated: 2023-04-14
+description: How to record audio or video from a tab, window or screen
+---
+
+APIs such as [chrome.tabCapture][tabcapture] and [getDisplayMedia][get-display-media] provide a
```

Comment:
Suggestion: rephrase `a variety of ways to` which is very generic. 

"This guides explains different approaches for recording audio and video from a tab, window or a whole screen using APIs such as [chrome.tabCapture][tabcapture] or [getDisplayMedia][get-display-media]."